manim/mobject/types/vectorized_mobject.py

```diff
@@ -898,13 +892,7 @@ def rotate(
         return self
 
     def scale_handle_to_anchor_distances(self, factor: float):
-        """If the distance between a given handle point H and its associated
-        anchor point A is d, then it changes H to be a distances factor*d
-        away from A, but so that the line from A to H doesn't change.
-        This is mostly useful in the context of applying a (differentiable)
-        function, to preserve tangency properties.  One would pull all the
-        handles closer to their anchors, apply the function then push them out
-        again.
+        """If the distance between a given handle point H and its associated anchor point A is d, then it changes H to be a distances factor*d away from A, but so that the line from A to H doesn't change. This is mostly useful in the context of applying a (differentiable) function, to preserve tangency properties.  One would pull all the handles closer to their anchors, apply the function then push them out again.
```

Comment:
```suggestion
        """If the distance between a given handle point H and its associated anchor point A is d, then it changes H to be a distances factor*d away from A, but so that the line from A to H doesn't change. 
        
        This is mostly useful in the context of applying a (differentiable) function, to preserve tangency properties.  
        One would pull all the handles closer to their anchors, apply the function 
        then push them out again.
```
Also needs to be made shorter.